week6/src/Store.java

```diff
@@ -0,0 +1,202 @@
+
+        //adding some stuff to the shop
+        ArrayList<Animal> animalProducts = new ArrayList<>();
+        animalProducts.add(new Animal("Milk", 21, "20/08/2019", 1, 5));
+        animalProducts.add(new Animal("Eggs", 12, "23/08/2019", 0.05, 6));
+        animalProducts.add(new Animal("Cheese", 30, "21/08/2019", 2, 3));
+        animalProducts.add(new Animal("Sour cream", 45, "20/08/2019", 1, 5));
+
+        ArrayList<Vegetal> vegetalProducts = new ArrayList<>();
+        vegetalProducts.add(new Vegetal("Pumpkin", 23.5, "25/08/2019", 2, "B, C, A"));
+        vegetalProducts.add(new Vegetal("Carrots", 20, "21/09/2019", 1, "B2, C, A"));
+        vegetalProducts.add(new Vegetal("Onions", 15, "13/09/2019", 2, "C, A"));
+        vegetalProducts.add(new Vegetal("Potatoes", 14.5, "04/09/2019", 2, "B, C"));
+
+        ArrayList<Sales> salesList = new ArrayList<>();
+
+        BufferedReader ob = new BufferedReader(new InputStreamReader(System.in));
+        Scanner sc = new Scanner(System.in);
+        int choice;
+
+
+
+        //repeat main menu until [4] Exit
+        do {
+            System.out.println("----------------------------------------");
+            System.out.println("[1] Create a product and add it to stock\n" +
+                    "[2] Sell product\n" +
+                    "[3] Display daily sales report\n" +
+                    "[4] Exit");
+            System.out.println("----------------------------------------");
+            int tempChoice = sc.nextInt();
+            choice = tempChoice;
+
+            //second menu (animal, vegetal)
+            switch(choice) {
+
+                case 1:
+                    System.out.println("Product type\n" +
+                            "----------------------------------------\n" +
+                            "[1] Animal\n" +
+                            "[2] Vegetal");
+                    int secondChoice = sc.nextInt();
+
+                    switch (secondChoice) {
+
+                        case 1:
+                            //adding new animal product values
+                            System.out.println("Enter animal product: ");
+                            String a1 = ob.readLine();
+                            System.out.println("Enter price [eg. 23,6]: ");
+                            double a2 = sc.nextDouble();
+                            System.out.println("Enter expiration date [dd/mm/yyyy]: ");
+                            String a3 = ob.readLine();
+                            System.out.println("Enter weight in kilograms [eg. 35,5]: ");
+                            double a4 = sc.nextDouble();
+                            System.out.println("Enter storage temperature [eg. 7,6]: ");
+                            double a5 = sc.nextDouble();
+
+                            //creating the new animal product and adding it to the animal products list
+                            animalProducts.add(new Animal(a1, a2, a3, a4, a5));
```

Comment:
so you cannot add an animal product with a quantity? If you want to add 3 milk you need to call this 3 times?